build.sbt

```diff
@@ -118,7 +118,7 @@ libraryDependencies ++= Seq(
   "net.sourceforge.jtds" % "jtds" % "1.3.1" % "provided",
   "org.apache.derby" % "derby" % "10.11.1.1" % "provided",
   "org.xerial" % "sqlite-jdbc" % "3.21.0.1" % "test",
-  "org.json4s" %% "json4s-scalap" % "3.5.3",
+  "org.json4s" %% "json4s-scalap" % "3.5.3"
```

Comment:
why delete `,`?

Reply:
Leaving trailing comma when declaring the collection or sequence is not valid expression in Scala:
```
scala> Seq("a", "b")
res1: Seq[String] = List(a, b)

scala> Seq("a", "b",)
<console>:1: error: illegal start of simple expression
```
SBT for some reason will allow for this construct, but i.e. IntelliJ would warn you with an error. 

I believe the comma has been left by mistake when ScalaTest has been moved away from the block (due to lack of Scala/2.13.0-M4 support), since the previous version (with ScalaTest) didn't have a trailing comma. If there was a genuine reason why you wanted the comma there, please let me know.

Reply:
> SBT for some reason will allow for this construct

not only sbt. Scala 2.12.2 or later allow trailing comma
- https://docs.scala-lang.org/sips/completed/trailing-commas.html
- https://github.com/scala/scala/commit/d6837aef059b6d2df1eac10284036ec1ed603a7b

Reply:
I believe IntelliJ support tailing comma
- https://youtrack.jetbrains.com/issue/SCL-10597
- https://github.com/JetBrains/intellij-scala/commit/1ae11156a96ef29dc1cb03dd7ff927e321dc574f
- https://github.com/JetBrains/intellij-scala/commit/6ad57b6775db910ca8b6bd8546a6da7fce43fa48

Reply:
Thanks for the detailed info. For some reason my IntelliJ 2018.1.4 (Comunity Edition) does seem to report this as a break:

![trailing_commas_intellij](https://user-images.githubusercontent.com/5698083/40703912-a7f84c78-63de-11e8-8600-cbe56f273729.png)

However, given it should not been the case I have reverted my change to trailing comma of build.sbt and will investigate further. The change is about replacing `assertEquals` method with ScalaTest matchers so no reason for the trailing comma discussion to block the review.